go.mod

```diff
@@ -1,14 +1,126 @@
 module github.com/decred/decrediton
 
-go 1.16
+go 1.18
 
 require (
-	decred.org/dcrdex v0.4.3
+	decred.org/dcrdex v0.5.0-rc2.0.20220819154427-0dee62f8587b
```

Comment:
Will put this on the final `v0.5.0` tag when it's created, but it looks like it will be this commit.